### DISCUSSION TITLE
SteamVR Background for QuestAppLauncher

### DISCUSSION THREAD

**Discussion by reloxx13** [reloxx13/QuestAppLauncher_Assets#4] [Show and tell]
You can use the nice SteamVR background for the QuestAppLauncher.    
   
You need to install Steam and SteamVR.   
After that you can find the background images in your steams app dir e.g.:
`C:\Program Files (x86)\Steam\SteamApps\common\SteamVR\resources`   

Some more backgrounds are found in this workshop:   
[Steam Workshop: Steam VR backgrounds](https://steamcommunity.com/sharedfiles/filedetails/?id=691904438)   
After adding them, they are downloaded in `C:\Program Files (x86)\Steam\steamapps\workshop\content\250820`   
   
Copy the backgrounds to your QuestAppLauncher directory on your Headset
`Android/data/aaa.QuestAppLauncher.App/files/backgrounds`

Due copyrights i cannot upload them here.

**Comment by WilsonBradley** [comment]
Thanks.. this part works great..

**Reply by WilsonBradley** [reply]
How to get icons show up on Quest 2.. Mine is blank.. :-(
